eos/references.yaml

```diff
@@ -1370,6 +1392,13 @@ LHCb:2021D:
     id: oai:arXiv.org:2107.13428
   inspire-id: LHCb:2021xxq
   title: Angular analysis of the rare decay $B_s^0\to\phi\mu^+\mu^-$
+LMPW:2015A:
+  authors: Lang, C. B. and Mohler, Daniel and Prelovsek, Sasa and Woloshyn, R. M.
+  eprint:
+    archive: arXiv
+    id: oai:arXiv.org:1501.01646
+  inspire-id: Lang:2015hza
+  title: Predicting positive parity B$_s$ mesons from lattice QCD
```

Comment:
$B_s$ surely?

Reply:
That was the official Inspire name, but I agree that $B_s$ looks nicer, I changed it